### DISCUSSION TITLE
How to solve this rendering error in VSCode?

### DISCUSSION THREAD

**Discussion by juliohm** [quarto-dev/quarto-cli#6469] [Q&A] [answered]
### Description

I am trying to create a book with Quarto for the first time in VSCode, but the rendering button is showing this error:

![image](https://github.com/quarto-dev/quarto-cli/assets/3345261/0ad5c48c-fcba-495a-91da-be0943762c1b)

I am on VSCode:

Version: 1.67.2
Commit: c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5
Date: 2022-05-17T18:23:40.286Z
Electron: 17.4.1
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Linux x64 5.15.122-1-MANJARO

Any idea on how I could solve this error message? It seems to be related to my Jupyter installation somehow. Maybe the version is too new for the VSCode Quarto extension? Any help is appreciated.

**Comment by mcanouil** [comment, marked as answer]
You should update VSCode and extensions.

**Comment by juliohm** [comment]
You are 100% correct @mcanouil. Fixed the issue by updating to the latest VSCode 1.81.

**Reply by mcanouil** [reply]
Glad this was it.
